doc/api.md

```diff
@@ -676,3 +676,7 @@ culminates in chain loading from the Razor server)
 The URL accepts the parameter `nic_max` which should be set to the maximum
 number of network interfaces that respond to DHCP on any given machine. It
 defaults to 4.
+
+If this request occurs over HTTPS, the `http_port` parameter must be supplied
```

Comment:
Would probably be good to spell out what goes into the http_port. Right now, this is more a justification of why we need to know that - for the user, it's more important to know what information to supply.